manifests/graphite_reqs.pp

```diff
@@ -49,8 +46,7 @@
   }
   package { 'python-sqlite3dbm':
     ensure => '0.1.4-6.el7',
-    provider => 'pip',
-    require => [Package['python-pip'],Exec['/bin/pip install requests[security]']],
+    require => Exec['/bin/pip install requests[security]'],
```

Comment:
Yeah, sqlite3dbm won't install without the security extras set of the requests package.